frontend/src/metabase/static-viz/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { scaleLinear } from "@visx/scale";
+import { Group } from "@visx/group";
+import { ClipPath } from "@visx/clip-path";
+import { formatNumber } from "../../lib/numbers";
+import { Text } from "../Text";
+import { Pointer } from "./Pointer";
+import { CheckMarkIcon } from "./CheckMarkIcon";
+import { createPalette, getBarText, getColors } from "./utils";
+import { ProgressBarData } from "./types";
+
+const layout = {
+  width: 400,
+  height: 110,
+  margin: {
+    top: 40,
+    right: 100,
+    left: 20,
+  },
+  barHeight: 40,
+  borderRadius: 5,
+  labelsMargin: 16,
+  iconSize: 20,
+  pointer: {
+    width: 20,
+    height: 10,
+  },
+};
+
+interface ProgressBarProps {
```

Comment:
AFAIK the BE interface for static viz is `{ data, labels, settings }`. Have you confirmed the API with a BE dev?

Reply:
yeah, we can futz w/ arity in the BE if you want, that seems awkward as is

Reply:
Fixed that, thanks